scripts/superdesk-search/react.js

```diff
@@ -1576,6 +1582,27 @@ angular.module('superdesk.search.react', [
                         }, 500, false);
                     },
 
+                    dbClick: function(item) {
+                        var activities = superdesk.findActivities({action: 'list', type: 'archive'}, item);
+
+                        var canEdit = _.reduce(activities, function(result, value, key) {
```

Comment:
No need to change this, but just FYI, it can also be:
```js
var canEdit = _.reduce(activities, (result, value) => result || value._id === 'edit.item', false);
```
Also, your build might fail because of using double quotes `"`